src/tooltip.d.ts

```diff
@@ -1,104 +1,111 @@
-import React from 'react';
-import { GestureResponderEvent, StyleProp, ViewStyle } from 'react-native'
+// Type definitions for react-native-walkthrough-tooltip 0.5.3
+// Definitions by: Siraj Alam https://github.com/sirajalam049
 
+declare module 'react-native-walkthrough-tooltip' {
 
-export interface ToolTipSize {
-    width: number,
-    height: number
-}
 
+    import React from 'react';
+    import { GestureResponderEvent, StyleProp, ViewStyle } from 'react-native'
 
-export interface ToolTopRect {
-    x: number,
-    y: number,
-    width: number,
-    height: number,
-}
 
-/**
- * Style Props
- * The tooltip styles should work out-of-the-box for most use cases, 
- * however should you need you can customize the styles of the tooltip using these props.
- */
-export interface TooltipStyleProps {
+    export interface ToolTipSize {
+        width: number,
+        height: number
+    }
 
-    // Styles the triangle that points to the called out element
-    arrowStyle?: StyleProp<ViewStyle>
 
-    // Styles the overlay view that sits behind the tooltip, but over the current view
-    backgroundStyle?: StyleProp<ViewStyle>
+    export interface ToolTopRect {
```

Comment:
noticed a typo here, should be `ToolTipRect` ?

Reply:
yep, hold my bear for a minute.